src/site/content/en/blog/excalidraw-and-fugu/index.md

```diff
@@ -0,0 +1,466 @@
+});*/
+}
+```
+
+The secret for making this happen is by calling getAsFileSystemHandle on the data transfer item when
+the File System Access API is supported. I then pass this file handle to loadFromBlob, which you may
+remember from a couple slides ago. So many things you can do with files: opening, saving,
+over-saving, dragging, dropping. My colleague Pete and I have documented all these tricks and more
+in [our article](/file-system-access/) so you can catch up in case all this went a little too
+fast.
+
+Another system integration currently on Android, Chrome OS, and Windows is through the Web Share
+Target API. Here I am in the Files app in my Downloads folder. I can see two files, one of them with
+the non-descript name untitled and a timestamp. To check what it contains, I click on the three
+dots, then share, and one of the options that appears is Excalidraw. When I tap the icon, I can then
+see that the file just contains the I/O logo again.
+
+{% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/S3mqjw6bM7y28WxSpd0y.mov", autoplay=true, muted=true, playsinline=true %}
+
+One thing you can do with files that I haven't talked about yet is doubleclick them. What typically
+happens when you doubleclick a file is that the app that's associated with the file's MIME type
+opens. For example for .docx this would be Microsoft Word.
+
+Excalidraw used to have an Electron version of the app that supported such file type associations,
+so when you double-clicked an .excalidraw file, the Excalidraw Electron app would open. Lipis, whom
+you have already met before, was both the creator and the deprecator of Excalidraw Electron. I asked
+him why he felt it was possible to deprecate the Electron version.
+
+> People have been asking for an Electron app since the beginning, mainly because they wanted to
+> open files by double-clicking. We also intended to put the app in app stores. In parallel, someone
+> suggested creating a PWA instead, so we just did both. Luckily we were introduced to Project Fugu
+> APIs like file system access, clipboard access, file handling, and more. With a sole click you can
+> install the app on your desktop or mobile, without the extra weight of Electron. It was an easy
+> decision to deprecate the Electron version, concentrate just on the web app, and make it the
+> best-possible PWA. On top, we're now able to publish PWAs to the Play Store and the Microsoft
+> Store! That's huge!
+
+One could say Excalidraw for Electron was not deprecated because Electron is bad, not at all, but
+because the web has become good enough. I like this!
+
+When I say "the web has become good enough", it's because of features like the upcoming File
+Handling. This is a regular macOS Big Sur installation. Now check out what happens when I
+right-click an Excalidraw file. I can choose to open it with Excalidraw, the installed PWA. Of
+course double-clicking would work, too, it's just less dramatic to demonstrate in a screencast.
```

Comment:
>it's just less dramatic to demonstrate in a screencast.

Stuff like this makes me think you should included some screen shots in this article. I don't want to hold up this PR though. If you do, we can merge it later.